perun-core/src/test/java/cz/metacentrum/perun/core/entry/FacilitiesManagerEntryIntegrationTest.java

```diff
@@ -235,6 +236,40 @@ public void getFacilitiesByAttribute() throws Exception {
 		assertTrue("results must contain user", facilitiesManagerEntry.getFacilitiesByAttribute(sess, attributeName, "value").contains(facility));
 	}
 
+	@Test
+	public void getFacilitiesByAttributeWithAttributes() throws Exception {
+		System.out.println(CLASS_NAME + "getFacilitiesByAttributeWithAttributes");
+
+		// Check if the attribute already exists
+		Attribute attr;
```

Comment:
would be nice to reuse some setUp method, for example `Attribute attr = setUpAttribute1(); `, and move the try-catch block to the setUp method instead